avr/bootloaders/HoodLoader2/Board/Board.h

```diff
@@ -57,6 +57,7 @@ extern "C" {
 			#define ARDUINO_MICRO_PID   		0x0037 // Bootloader, not program!
 			#define ARDUINO_DUE_PID             0x003D
 			
+			#ifndef USB_DESCRIPTOR_STRING // If the Makefile did not supply a product string...
```

Comment:
This allows `vidpid.txt` to redefine the display string to whatever the user want to.